build-image/first-boot.sh

```diff
@@ -86,14 +92,9 @@ until ping -c1 9.9.9.9 &>/dev/null || [ "$(wget -qO- http://www.msftncsi.com/ncs
   if [ $cnt -eq 100 ]; then
     echo "FAILED"
     if grep -q "openHABian" /etc/wpa_supplicant/wpa_supplicant.conf && iwconfig 2>&1 | grep -q "ESSID:off"; then
-      echo ""
-      echo "I was not able to connect to the configured Wi-Fi."
-      echo "Please check your signal quality. Reachable Wi-Fi networks are:"
+      echo -n "$(timestamp) [openHABian] I was not able to connect to the configured Wi-Fi. \n Please check your signal quality. Reachable Wi-Fi networks are: \n"
       iwlist wlan0 scanning | grep "ESSID" | sed 's/^\s*ESSID:/\t- /g'
-      echo ""
-      echo "Please try again with your correct SSID and password."
-      echo "The following Wi-Fi configuration was used:"
-      echo ""
+      echo -n "$(timestamp) [openHABian] Please try again with your correct SSID and password. \n The following Wi-Fi configuration was used: \n"
```

Comment:
You did change the number of newlines in those two blocks. They had their esthetic purpose. I'd suggest to restore them.